ament_cmake_auto/cmake/ament_auto_find_build_dependencies.cmake

```diff
@@ -27,18 +32,35 @@
 # @public
 #
 macro(ament_auto_find_build_dependencies)
-  if(ARGN)
+  cmake_parse_arguments(ARG "" "" "REQUIRED_PACKAGES" ${ARGN})
```

Comment:
OK, I'll update this. I want to point out that there's a lot of existing code that doesn't do this, in case it is of significant enough concern that we should update that code as well.